README.md

```diff
@@ -1,6 +1,381 @@
 <p align="center"><img width="280" src="https://s3.eu-central-1.amazonaws.com/assets.beatswitch.com/distil-logo.svg"></p>
 <p align="center">
-    <a href="https://travis-ci.org/beatswitch/distil"><img src="https://img.shields.io/travis/BeatSwitch/distil/master.svg?style=flat-square" alt="Build Status"></a>
+    <a href="https://travis-ci.org/BeatSwitch/distil"><img src="https://img.shields.io/travis/BeatSwitch/distil/master.svg?style=flat-square" alt="Build Status"></a>
     <a href="https://styleci.io/repos/102117231/shield"><img src="https://styleci.io/repos/102117231/shield" alt="StyleCI"></a>
     <a href="license.md"><img src="https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square" alt="Licence"></a>
 </p>
+
+Distil provides a simple API to describe a variable number of conditions that should be met by a result set in PHP. This, for example, makes dynamically constructing queries through small value objects a breeze.
+
+- [Getting Started](#getting-started)
+    - [Simple Example](#simple-example)
+    - [Installation](#installation)
+    - [Requirements](#requirements)
+- [Concepts](#concepts)
+    - [Criteria](#criteria)
+    - [Criterion](#criterion)
+    - [Keywords](#keywords)
+    - [Common Criteria](#common-criteria)
+    - [Factories](#factories)
+
+
+# Getting Started
+
+## Simple Example
+
+Let's consider the following scenario: you have a query object `GetPosts` that returns all posts of your blog by default. However, you only need to retrieve posts from a specific author on some places. 
+The query itself is identical, you only need to strip out the ones that were created by anyone other than the given author. To do so, the query object can accept criteria that may or may not hold the "author" criterion:
+```php
+$authorId = 369;
+
+// Author is an implementation of Distil\Criterion which acts as a Criteria factory.
+$criteria = Author::criteria($authorId);
+
+$posts = (new GetPosts())->get($criteria);
```

Comment:
Rewrite this to:

```php
$posts = $getPostsQuery->get($criteria);
```

It doesn't matter how the `$getPostsQuery` is instantiated in this example. Besides `new GetPosts()` is unlogic and probably just confuses the user.